src/main/java/edu/cornell/kfs/fp/document/validation/impl/CuDisbursementVoucherDocumentPreRules.java

```diff
@@ -1,67 +1,242 @@
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU Affero General Public License for more details.
- * 
+ *
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 package edu.cornell.kfs.fp.document.validation.impl;
 
+import java.util.ArrayList;
+import java.util.List;
+
 import org.apache.commons.lang.StringUtils;
+import org.apache.commons.lang.WordUtils;
 import org.kuali.kfs.fp.document.DisbursementVoucherDocument;
 import org.kuali.kfs.fp.document.validation.impl.DisbursementVoucherDocumentPreRules;
+import org.kuali.kfs.pdp.PdpParameterConstants;
+import org.kuali.kfs.pdp.businessobject.PaymentNoteText;
 import org.kuali.kfs.sys.KFSConstants;
 import org.kuali.kfs.sys.businessobject.PaymentSourceWireTransfer;
 import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.kfs.sys.service.impl.KfsParameterConstants;
 import org.kuali.rice.core.api.config.property.ConfigurationService;
+import org.kuali.rice.core.api.util.type.KualiInteger;
 
+import edu.cornell.kfs.fp.CuFPConstants;
+import edu.cornell.kfs.fp.document.CuDisbursementVoucherConstants;
 import edu.cornell.kfs.sys.CUKFSKeyConstants;
 
+import org.kuali.rice.coreservice.framework.parameter.ParameterService;
+import org.kuali.rice.krad.document.Document;
+
 /**
  * Checks warnings and prompt conditions for dv document.
  */
 public class CuDisbursementVoucherDocumentPreRules extends DisbursementVoucherDocumentPreRules {
 
-    /**
-     * This method returns true if the state of all the tabs is valid, false otherwise.
-     *
-     * @param dvDocument submitted disbursement voucher document
-     * @return Returns true if the state of all the tabs is valid, false otherwise.
-     */
-    @SuppressWarnings("deprecation")
-    protected boolean checkWireTransferTabState(DisbursementVoucherDocument dvDocument) {
-        boolean tabStatesOK = true;
-
-        PaymentSourceWireTransfer dvWireTransfer = dvDocument.getWireTransfer();
-
-        // if payment method is CHECK and wire tab contains data, ask user to clear tab
-        if ((StringUtils.equals(KFSConstants.PaymentSourceConstants.PAYMENT_METHOD_CHECK, dvDocument.getDisbVchrPaymentMethodCode()) || StringUtils.equals(KFSConstants.PaymentSourceConstants.PAYMENT_METHOD_DRAFT, dvDocument.getDisbVchrPaymentMethodCode())) && hasWireTransferValues(dvWireTransfer)) {
-            String questionText = SpringContext.getBean(ConfigurationService.class).getPropertyValueAsString(CUKFSKeyConstants.QUESTION_CLEAR_UNNEEDED_WIRE_TAB);
-
-            boolean clearTab = super.askOrAnalyzeYesNoQuestion(KFSConstants.DisbursementVoucherDocumentConstants.CLEAR_WIRE_TRANSFER_TAB_QUESTION_ID, questionText);
-            if (clearTab) {
-                // NOTE: Can't replace with new instance because Foreign Draft uses same object
-                clearWireTransferValues(dvWireTransfer);
+  /**
```

Comment:
This file appears to be using 2 spaces per level of indentation instead of 4. I would suggest changing it to 4 spaces per level so that it's consistent with our other code files.